tree/tree/inc/TTreeCache.h

```diff
@@ -77,10 +129,14 @@ class TTreeCache : public TFileCacheRead {
    virtual Int_t        DropBranch(const char *branch, Bool_t subbranches = kFALSE);
    virtual void         Disable() {fEnabled = kFALSE;}
    virtual void         Enable() {fEnabled = kTRUE;}
+   void                 SetOptimizeMisses(bool opt);
```

Comment:
In this file/section the routine are (and thus should stay) listed alphabetically.